docs/msiv1_token_revocation.md

```diff
@@ -0,0 +1,86 @@
+# MSAL Support for MSI v1 Token Revocation and Capabilities
+
+---
+
+## Goals
+
+1. App developers and higher level SDKs like Azure SDK rely on the [CAE protocol](https://learn.microsoft.com/en-us/entra/identity-platform/app-resilience-continuous-access-evaluation?tabs=dotnet) for token revocation scenarios (`WithClaims`, `WithClientCapabilities`). 
+1. RPs are enabled to perform token revocation.
+1. A telemetry signal for eSTS exists to differentiate which apps are CAE enlightened.
+
+## Flow diagram - revocation event
+
+```mermaid
+sequenceDiagram
+    participant Resource
+    actor CX
+    participant MSAL
+    participant SF        
+    participant eSTS
+
+rect rgb(173, 216, 230)
+    CX->> Resource: 1. Call resource with "bad" token T
+    Resource->>CX: 2.HTTP 401 + claims C
+    CX->>CX: 3. Parse response, extract claims C
+    CX->>MSAL: 4. MSI.AcquireToken <br/> WithClaims(C) <br/> WithClientCapabilities("cp1")
+end
+rect rgb(215, 234, 132)
+    MSAL->>MSAL: 5. Get token T from cache. Assume it is the "bad" token.
+    MSAL->>SF: 6. Call MITS_endpoint?xms_cc=cp1&revoked_token=SHA256(T)
```

Comment:
```suggestion
    MSAL->>SF: 6. Call MITS_endpoint?claims=C&token_sha256_to_refresh=SHA256(T)
```

Reply:
How will this work Ray? Claims and client capabilities need to be passed separately, because claims bypass cache and cp1 doesn't.